x/examples/outline-connectivity/main.go

```diff
@@ -150,6 +153,37 @@ func main() {
 			if err != nil {
 				log.Fatalf("Failed to output JSON: %v", err)
 			}
+			// Send error report to collector if specified
+			if !success && *collectorFlag != "" {
+				jsonData, err := json.Marshal(record)
```

Comment:
We should work on a more formal format at some point. Perhaps we start a document to discuss that.

Reply:
do you mean the JSON format that holds the error flags?

Reply:
Yes. We may need to nest that into a report object with metadata, and we may want to send multiple reports in one request as well.
We also need to think of other use cases, like proxyless solutions.

Reply:
The collector just expects a JSON object. When received, it flattens it and also creates new column headers for previously unseen values/tags. So I am not enforcing any format on the server side (at some point we can also consider sanitizing & serializing the JSON). We can nest various reports into one parent JSON. All this can be done on the client side when generating the report...